libs/ui/src/components/NeoNftCard/NFTMediaInfo.vue

```diff
@@ -12,11 +12,12 @@
       <CollectionDetailsPopover
         v-if="nft.collection.name || nft.collection.id"
         :show-delay="collectionPopoverShowDelay"
+        class="is-size-7 nft-info-collection-name is-ellipsis"
         :nft="nft">
         <template #content>
           <nuxt-link
```

Comment:
nuxt-link in `libs/ui` 😢

Reply:
this clearly need rework as it depend on some stuff that is not inside libs/ui

Reply:
1. agree
2. CollectionDetailsPopover isn't new, it's been there since February : https://github.com/kodadot/nft-gallery/commit/5b59514de9baf102a929b0a0f187592ff4b15e58
3. ill change nuxt-link to `<a/>`